src/frontend/src/scheduler/local.rs

```diff
@@ -79,7 +81,8 @@ impl LocalQueryExecution {
         self.epoch = Some(epoch);
         let plan_fragment = self.create_plan_fragment()?;
         let plan_node = plan_fragment.root.unwrap();
-        let executor = ExecutorBuilder::new(&plan_node, &task_id, context, epoch);
+        let vnode_ranges = full_vnode_range();
+        let executor = ExecutorBuilder::new(&plan_node, &task_id, &vnode_ranges, context, epoch);
```

Comment:
This builder is root task executed in frontend, so we should not have any vnodes